libs/brackets-server/embedded-ext/livepreview/main.js

```diff
@@ -195,7 +195,8 @@ define(function (require, exports, module) {
             href: "#"
         })
         .css({
-            display: "none"
+            display: "none",
+            visibility: PreferencesManager.getViewState("projectType") === "demo" ? "hidden" : "visible"
```

Comment:
setting visibility hidden leaves empty area above the icon for live preview in demo mode
what looks strange for first time WATT-demo user (empty area above icon)

How about using different live preview icon (e.g. play icon taken from DE)
and placing it on the first place.
Also not sure why we can't use display: "none" attribute

Reply:
Ok, I understood that it must be hidden. Now Live preview button is not added for remote samples.